src/components/Menu/useMenu.ts

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useSelector, useDispatch } from "react-redux";
+import {useEffect,useState} from 'react'
+import {actionsCategories,selectorsCategories} from 'store/sliceCategories'
+import { AppDispatch } from 'store/store';
+import type { DrawerProps, } from 'antd';
+
+
+export const useMenu= () => {
+  const [visible, setVisible] = useState(false);
+  const [placement, setPlacement] = useState<DrawerProps['placement']>('left');
+  const showDrawer = () => {
+    setVisible(true);
+  };
+  const onClose = () => {
```

Comment:
```suggestion
  const hideDrawer = () => {
```
если функция показать называется showDrawer, то логично также и обратную назвать. Чем больше единообразия в коде - тем лучше